lib/rouge/lexers/mason.rb

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Mason < TemplateLexer
+      title 'Mason'
+      desc 'HTML::Mason'

```

Comment:
Could you change this to 'The HTML::Mason framework (metacpan.org/pod/HTML::Mason)'?

Reply:
Done!